packages/subgraph/src/v1-analytics/mapping.ts

```diff
@@ -31,9 +31,16 @@ export function handleLiquidityAdded(event: LiquidityAdded): void {
     assetBalance.assetId = event.params.assetId;
     assetBalance.router = router.id;
     assetBalance.amount = new BigInt(0);
+    assetBalance.supplied = new BigInt(0);
+    assetBalance.locked = new BigInt(0);
   }
   // add new amount
   assetBalance.amount = assetBalance!.amount.plus(event.params.amount);
+
+  // add invested
+  assetBalance.supplied = assetBalance.supplied.plus(event.params.amount);
```

Comment:
nit: needs an explicit unwrap above and none here?

Reply:
i dont think it does -- in fact i think the one above is unnecessary since its assigned in the case where `assetBalance` does not exist